C-Plus-Plus/stack/clone_a_stack_without_usinig_extra_space.cpp

```diff
@@ -0,0 +1,55 @@
+// Below is the implementation of
+// Cloning a stack into destination stack without using extra space
+#include<bits/stdc++.h>
+using namespace std;
+// This Function takes takes two stakes as input and
+// copy data of first stack to the second stack  
+
```

Comment:
Remove the redundant comments instead write one line about your function description